config/locales/server.en.yml

```
@@ -9,6 +9,8 @@ en:
     show_filter_by_solved_status: "Show a dropdown to filter a topic list by solved status."
     notify_on_staff_accept_solved: "Send notification to the topic creator when a post is marked as solution by a staff."
     disable_solved_education_message: "Disable education message for solved topics."
+    accept_solutions_topic_starter: "Allow the topic starter to accept a solution."
+    solved_add_schema_markup: "When to add QAPage schema markup to HTML?"
```

```suggestion
    solved_add_schema_markup: "Add QAPage schema markup to HTML"
```
The original text reads like a question to me and I believe this is more clear.

---

Good point. Fixed.